pkg/test/framework/features/features.yaml

```diff
@@ -58,3 +58,7 @@ features:
   extensibility:
   # features releated to the lifecycle of istio installations
   installation:
+    multicluster:
+      central-istiod:
+      mutlimaster:
```

Comment:
multimaster?

Reply:
Is this what we are calling it in the docs?

Reply:
@howardjohn These were the instructions I received regarding features:

> test and release group will give feedback on the top ~2 levels.  Below that, it's really up to the relevant working group how they want to think about their feature organization

I picked the name solely based on the name in the subdirectory.  In terms of structure - I placed these 3 under `installation` because they are just standard reachability testing.  If I wanted to added a test with basic version based routing (in https://github.com/istio/istio/pull/24503 for example) would this go under `installation.multicluster.central-istio.routing`?  Or should new sub labels get added in `traffic.routing.multicluster.central-istio`?

Reply:
@stevenctl - any thoughts on how features should be named/structured?

Reply:
Putting them under installation makes the most sense to me. The other features exist outside of the multicluster scenario.

Reply:
@howardjohn should I just leave the `multimaster` and `remote` tests as `installation.multicluster` for the feature label?